sdk/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "@testkube/sdk",
+  "version": "0.1.3",
+  "description": "Typescript SDK for running Testkube tests and creating pipelines.",
+  "main": "dist/testkube.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

There should be `build` script to build with `tsc` and generate client with `oazapfts`. Additionally, `prepublish` script should call it, so `npm publish` would build it correctly.

```suggestion
  "scripts": {
    "build": "oazapfts ../api/v1/testkube.yaml ./src/generated-client.ts && tsc",
    "prepublish": "npm run build",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
```